spec.bs

```diff
@@ -2040,6 +2045,9 @@ To <dfn>build trusted bidding signals url</dfn> given a [=URL=] |signalsUrl|, an
   1. [=list/Append=] [=serialize an integer|serialized=] |experimentGroupId| to |queryParamsList|.
 1. Let |fullSignalsUrl| be |signalsUrl|.
 1. Set |fullSignalsUrl|'s [=url/query=] to the result of [=string/concatenating=] |queryParamsList|.
+
+  Note: Not using [=urlencoded serializer|application/x-www-form-urlencoded serializer=] to
+    construct the query string because it uses a different percent encode set from the implementation.

```

Comment:
This should eventually give a better reason (e.g. that `application/x-www-form-urlencoded` doesn't do the comma-separated-value thing), but for now we can just word it so it admits the possibility of non-Chrome implementations:

```suggestion
    construct the query string because it uses a different percent encode set from the Chrome implementation.
```